After tapping "+", new language caption+description pair difficult to access

When the "+" button is tapped, a new language (caption+description fields couple) is added at the bottom, but somehow it does not seem to always appear immediately, especially from the third language pair, I have to tap around a bit before being able to see it, not sure why.
![Screenshot_20200316-231105_Commons](https://user-images.githubusercontent.com/99590/76766756-d5c24980-67db-11ea-83a9-25b5a8a24acd.jpg)

And if I straightly go back to sleep from here, the app will crash.
```
D/Mbgl-HttpRequest: [HTTP] Request with response = 304: Not Modified
D/Mbgl-HttpRequest: [HTTP] Request with response = 304: Not Modified
D/Mbgl-HttpRequest: [HTTP] Request with response = 304: Not Modified
E/Choreographer: skip doframe for fr.free.nrw.commons.beta
D/AndroidRuntime: Shutting down VM
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: fr.free.nrw.commons.beta, PID: 25189
    java.lang.RuntimeException: Unable to resume activity {fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadActivity}: java.lang.IllegalStateException: Not allowed to start service Intent { act=fr.free.nrw.commons.upload.upload cmp=fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadService }: app is in background uid UidRecord{9e79b1a u0a115 TPSL bg:+4m0s113ms idle change:cached procs:1 seq(0,0,0)}
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3961)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3993)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1934)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6940)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalStateException: Not allowed to start service Intent { act=fr.free.nrw.commons.upload.upload cmp=fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadService }: app is in background uid UidRecord{9e79b1a u0a115 TPSL bg:+4m0s113ms idle change:cached procs:1 seq(0,0,0)}
        at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1595)
        at android.app.ContextImpl.startService(ContextImpl.java:1550)
        at android.content.ContextWrapper.startService(ContextWrapper.java:664)
        at fr.free.nrw.commons.upload.UploadController.prepareService(UploadController.java:81)
        at fr.free.nrw.commons.repository.UploadRemoteDataSource.prepareService(UploadRemoteDataSource.java:82)
        at fr.free.nrw.commons.repository.UploadRepository.prepareService(UploadRepository.java:67)
        at fr.free.nrw.commons.upload.UploadPresenter.onAttachView(UploadPresenter.java:130)
        at fr.free.nrw.commons.upload.UploadPresenter.onAttachView(UploadPresenter.java:25)
        at fr.free.nrw.commons.upload.UploadActivity.onResume(UploadActivity.java:192)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1514)
        at android.app.Activity.performResume(Activity.java:7311)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3953)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3993) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1934) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6940) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
E/ACRA: ACRA caught a RuntimeException for fr.free.nrw.commons.beta
    java.lang.RuntimeException: Unable to resume activity {fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadActivity}: java.lang.IllegalStateException: Not allowed to start service Intent { act=fr.free.nrw.commons.upload.upload cmp=fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadService }: app is in background uid UidRecord{9e79b1a u0a115 TPSL bg:+4m0s113ms idle change:cached procs:1 seq(0,0,0)}
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3961)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3993)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1934)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6940)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalStateException: Not allowed to start service Intent { act=fr.free.nrw.commons.upload.upload cmp=fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadService }: app is in background uid UidRecord{9e79b1a u0a115 TPSL bg:+4m0s113ms idle change:cached procs:1 seq(0,0,0)}
        at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1595)
        at android.app.ContextImpl.startService(ContextImpl.java:1550)
        at android.content.ContextWrapper.startService(ContextWrapper.java:664)
        at fr.free.nrw.commons.upload.UploadController.prepareService(UploadController.java:81)
        at fr.free.nrw.commons.repository.UploadRemoteDataSource.prepareService(UploadRemoteDataSource.java:82)
        at fr.free.nrw.commons.repository.UploadRepository.prepareService(UploadRepository.java:67)
        at fr.free.nrw.commons.upload.UploadPresenter.onAttachView(UploadPresenter.java:130)
        at fr.free.nrw.commons.upload.UploadPresenter.onAttachView(UploadPresenter.java:25)
        at fr.free.nrw.commons.upload.UploadActivity.onResume(UploadActivity.java:192)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1514)
        at android.app.Activity.performResume(Activity.java:7311)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3953)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3993) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1934) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6940) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
W/ACRA: Warning: Acra may behave differently with a debugger attached
```

--- a/app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java
+++ b/app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java
@@ -188,6 +188,7 @@ public class UploadMediaDetailFragment extends UploadBaseFragment implements
         UploadMediaDetail uploadMediaDetail = new UploadMediaDetail();
         uploadMediaDetail.setManuallyAdded(true);//This was manually added by the user
         uploadMediaDetailAdapter.addDescription(uploadMediaDetail);
+        rvDescriptions.scrollToPosition(uploadMediaDetailAdapter.getItemCount()-1);
     }
 
     @Override

--- a/app/src/main/res/layout/fragment_upload_media_detail_fragment.xml
+++ b/app/src/main/res/layout/fragment_upload_media_detail_fragment.xml
@@ -19,11 +19,6 @@
         android:layout_margin="@dimen/dimen_10"
         android:elevation="@dimen/cardview_default_elevation">
 
-        <androidx.core.widget.NestedScrollView
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:fillViewport="true">
-
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
@@ -139,8 +134,6 @@
 
                 </LinearLayout>
             </LinearLayout>
-        </androidx.core.widget.NestedScrollView>
-
 
     </androidx.cardview.widget.CardView>
 </RelativeLayout>

